assets/src/application/services/toaster/Toaster.tsx

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
+import { theme } from '@chakra-ui/core';
+
+import { Check, CloseCircleOutlined, InfoCircleOutlined, Spinner } from '@appDisplay/icons/svgs';
 
 import { ToasterProps } from './types';
 
-const Toaster: React.FC<ToasterProps> = ({ icon, message = 'loading...' }) => {
+const { colors } = theme;
+const fontSize = '1.2rem';
+
+const Toaster: React.FC<ToasterProps> = ({ message = 'loading...', type }) => {
+	const iconMapping = {
+		error: <CloseCircleOutlined color='rgb(255, 77, 79)' fontSize={fontSize} />,
+		info: <InfoCircleOutlined color={colors.blue['500']} fontSize={fontSize} />,
+		loading: <Spinner className='ee-loading-spinner' color={colors.cyan['500']} fontSize={fontSize} />,
+		success: <Check color={colors.green['500']} fontSize={fontSize} />,
+		warning: <InfoCircleOutlined color={colors.orange[500]} fontSize={fontSize} />,
+	};
```

Comment:
#### Great Idea 💯 

this could/should be extracted from this component though...
at least moved to the top of the file

we also need to figure out these theme colors soon and get them to match up with the rest of the EE color scheme

Reply:
agree, we need a centralized way to pull our colors